src/devcards/util/edn_renderer.cljs

```diff
@@ -69,8 +69,8 @@
    (keyword? x) (literal "keyword" x)
    (symbol? x)  (literal "symbol" x)
    (string? x)  (html-string x)
-   (map? x)     (html-map x)
-   (set? x)     (html-collection "set"    "#{" "}" x)
+   (map? x)     (html-map (into (sorted-map) x))
+   (set? x)     (html-collection "set"    "#{" "}" (into (sorted-set) x))
```

Comment:
The other option to sort the set is `(apply sorted-set x)` but there doesn't appear to be an advantage of one over the other so I favored `into` since that is how we sort the map and parallel code is just that much prettier.